typescript/sdk/src/router/ProxiedRouterDeployer.ts

```diff
@@ -50,13 +50,18 @@ export abstract class ProxiedRouterDeployer<
       await this.initializeArgs(chain, config),
     );
 
-    this.logger(`Transferring ownership of proxy admin to ${config.owner}`);
-    await super.runIfOwner(chain, proxyAdmin, () =>
-      this.multiProvider.handleTx(
-        chain,
-        proxyAdmin.transferOwnership(config.owner),
-      ),
-    );
+    await super.runIfOwner(chain, proxyAdmin, async () => {
+      this.logger(`Checking ownership of proxy admin to ${config.owner}`);
+
+      if ((await proxyAdmin.owner()) !== config.owner) {
```

Comment:
In what world isn't this the case? When deployer === config.owner?

Reply:
Yes indeed. On testnet, the deployer key is the owner. In those cases, we would just call transferOwnership despite the owner already being said correctly. (i.e. IQS and LL on testnet3 right now)